wikidatarefisland/services/schemaorg_property_mapper.py

```diff
@@ -0,0 +1,30 @@
+class SchemaorgPropertyMapper(object):
+    """ Side-step 2: Mapper of Wikidata properties and schema.org properties"""
+    def get_mapping(self):
+        raise NotImplementedError
+
+
+class WdqsSchemaorgPropertyMapper(SchemaorgPropertyMapper):
+    def __init__(self, wdqs_reader):
+        """
+
+        :type wdqs_reader: wikidatarefisland.data_access.WdqsReader
+        """
+        self.wdqs_reader = wdqs_reader
+        self._mapping = {}
+
+    def get_mapping(self):
+        if self._mapping == {}:
+            self._populate_mapping()
+        return self._mapping
+
+    def _populate_mapping(self):
+        query = """SELECT ?property ?url
+WHERE {
+  ?property wdt:P1628 ?url.
```

Comment:
I don't know huge amounts about it but I think https://www.python.org/dev/peps/pep-0498/#triple-quoted-f-strings might be the way to neatly do the injection of strings into here. I'd also probably in an ideal work like to see all the SPARQLs in separate template files

Reply:
I would say let's have jinja2 and templating (or a query builder?) Which one do you like?

I honestly quite dislike this current system